test/serializers/fieldset_test.rb

```diff
@@ -5,7 +5,7 @@ class Serializer
     class FieldsetTest < ActiveSupport::TestCase
       def test_fieldset_with_hash
         fieldset = ActiveModel::Serializer::Fieldset.new('post' => %w(id title), 'comment' => ['body'])
-        expected = { post: [:id, :title], comment: [:body] }
+        expected = { post: { id: {}, title: {} }, comment: { body: {} } }
```

Comment:
It be worth it to actually write logic/tests to allow specification of both attributes and relationships, rather than treat everything like a relationship